src/Products/GenericSetup/interfaces.py

```diff
@@ -535,9 +550,27 @@ def runAllImportStepsFromProfile(profile_id, purge_old=None,
         o Unless 'ignore_dependencies' is true this will also import
           all profiles this profile depends on.
 
+        o 'archive' is a tarball with files to import.
+          Note that metadata.xml is never read, so you cannot use dependencies.
+
```

Comment:
Why do you need that? Your argumentation (here and in a related PR) has been to facilitate upgrade specifications; but for this purpose, a folder is similar to an archive.

Reply:
'archive' has been supported for a long time. This merely adds some documentation to the interface.